doc/field_guide__lpv.md

```diff
@@ -0,0 +1,43 @@
+## Long Period Variables (lpv)
+Long Period Variables have periods over several tens of days.
+They incorporate various groups of cool luminous pulsators.
+The canonical example is Mira.
+
+### Classification and numbers
+- Supertypes
+  - variable
+  - periodic
+  - pulsator
+- Subtypes
+  - Miras
+  - Semiregulars
+  - L stars
+  - RV Tau
+- Occurrence rate: very common, about 10<sup>5</sup> expected in ZTF data
+

```

Comment:
Can you describe particular subtypes? What are L stars? I believe RV Tau stars belong to Type II Cepheids rather than "long period variables". (Type II Cepheids and LPVs follow different period-luminosity relations.) Long-period variables: https://ui.adsabs.harvard.edu/abs/2009AcA....59..239S/abstract, type II cepheids: https://ui.adsabs.harvard.edu/abs/2008AcA....58..293S/abstract

Reply:
I have updated the subtypes. And added the refs. I think we need to add a few figures to bring out the differences between the subtypes.

Reply:
Thanks Ashish. I think OSARGs also belong to "long-period variables" (basically, as a star evolves along the giant branch, the amplitude of pulsations grows, you start with low-amplitude oscillations (OSARGs), then the pulsation amplitude grows (SRVs --> Miras). It will be helpful to add a light curve of an SRV variable.

Reply:
Przemek, I have shifted OSARGs to LPVs with the comment that not everyone agrees.

Reply:
Thanks. Can you add an example light curve of a semiregular variable?

Reply:
Added one (for the first pair of coordinates that you gave).